### DISCUSSION TITLE
Adaptive styling

### DISCUSSION THREAD

**Discussion by kadenbadalian** [floating-ui/floating-ui#1328] [Q&A] [answered]
Noticed something with adaptive styling that seems off to me, and wanted to see what people thought. For something with placement of `bottom-end`, shouldn't it use the `top` and `right` attributes? Seems like currently it uses `top` and `left`.

**Comment by FezVrasta** [comment, marked as answer]
I think the option only takes in account the main placement. It'd be nice if the functionality was improved to handle what you are describing though.

**Reply by kadenbadalian** [reply]
Ahh ok that makes sense. Thanks!
